dymos/_options.py

```diff
@@ -11,8 +11,12 @@
 options.declare('include_check_partials', default=_icp_default, types=bool,
                 desc='If True, include dymos components when checking partials.')
 
-options.declare('plots', default='matplotlib', values=['matplotlib', 'bokeh'],
+options.declare('plots', default='bokeh', values=['matplotlib', 'bokeh'],
                 desc='The plot library used to generate output plots for Dymos.')
 
 options.declare('notebook_mode', default=False, types=bool,
                 desc='If True, provide notebook-enhanced plots and outputs.')
+
+options.declare('use_timeseries_prefix', default=True, types=bool,
+                desc='If True, prefix timeseries outputs with the variable type for states, times, controls,'
```

Comment:
Does it make sense to acknowledge here that the default behavior will change later on?

Reply:
Regarding the 1st and 3rd bullets, I will look into the DPI.  It's likely a font size issue but this will also impact your 3rd bullet point. _That_ issue can be alleviated by turning off the prefixes.

Regarding the number of columns in the plots, I will see if that can be done via the UI so the user can change it on the fly.

I'm planning on a relatively short time frame for the release of the next version, so I won't be making a note about the default behavior changing.